lesson01/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+  <html lang="en">
+    <head>
+      <title>Assignment 1</title>
+      <meta charset="utf-8">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+      <!-- jQuery -->
+      <script
+        src="https://code.jquery.com/jquery-3.5.1.min.js"
+        integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+        crossorigin="anonymous"></script>
+      <!-- Local Files -->
+      <script src="scripts.js"></script>
+      <link rel="stylesheet" type="text/css" href="stylesheet.css">
+      <link rel="preconnect" href="https://fonts.gstatic.com">
+      <link href="https://fonts.googleapis.com/css2?family=Karla&display=swap" rel="stylesheet">
+    </head>
+    <body>
+      <main>
```

Comment:
Though the focus is on the JS functions, you did an excellent job in displaying your HTML nicely!